tests/integration/kubernetes/k8s-confidential-attestation.bats

```diff
@@ -107,6 +116,84 @@ setup() {
 	[ "$status" -eq 1 ]
 }
 
+# this can run on all platforms
+@test "Cannot get CDH resource when affirming policy is set without reference values" {
+
+	# Require CPU0 to have affirming trust level. 
+	kbs_set_cpu0_resource_policy
+	kubectl apply -f "${K8S_TEST_YAML}"
+
+	# Retrieve pod name, wait for it to come up, retrieve pod ip
+	export pod_name=$(kubectl get pod -o wide | grep "aa-test-cc" | awk '{print $1;}')
+
+	# Check pod creation
+	kubectl wait --for=condition=Ready --timeout="$timeout" pod "${pod_name}"
+
+	sleep 5
+
+	kubectl logs aa-test-cc
+	cmd="kubectl logs aa-test-cc | grep -q aatest"
+	run $cmd
+	[ "$status" -eq 1 ]
```

Comment:
try to use the new pattern, see other tests here (changed after GPU attestation PR was merged)